inc/lock.class.php

```diff
@@ -731,6 +733,12 @@ static function deleteLock($item) {
          }
       }
 
+      // load general lock configuration
+      $db_lock = $DB->fetch_assoc(PluginFusioninventoryLock::getLock($item->fields['tablename'], 0));
```

Comment:
this 3 lines can be replaced by 
```php
$lockItemtype = PluginFusioninventoryLock::getLockFields($item->fields['tablename'], 0);
```

Also, i think it could be preferable to call the var `$generalLocks`

Reply:
done